pkg/controllers/webhookauthenticator/webhookauthenticator_controller.go

```diff
@@ -176,3 +164,61 @@ func (c *webhookAuthenticatorController) ensureKubeConfigSecret(ctx context.Cont
 	secret, _, err := resourceapply.ApplySecret(c.secrets, recorder, requiredSecret)
 	return secret, err
 }
+
+func (c *webhookAuthenticatorController) getAuthenticatorCertKeyPair() (key, cert []byte, err error) {
+	// waitingForCertKeyMsg lets us decide whether we should set progressing condition to true
+	// 1. nil: keep "progressing the same"
+	// 2. non-empty string: progress with the message in the string
+	// 3. empty string: stop progressing
+	var waitingForCertKeyMsg *string
+	defer func() {
+		if waitingForCertKeyMsg != nil {
+			cond := operatorv1.OperatorCondition{
+				Type:    "AuthenticatorCertKeyProgressing",
+				Status:  operatorv1.ConditionFalse,
+				Reason:  "AsExpected",
+				Message: "All is well",
```

Comment:
please use the same message as other operators for the sake of consistency

Reply:
This is actually it.

source:
https://github.com/openshift/cluster-authentication-operator/blob/f3ce0832d72f133018b14e6767c59e34c6fb2f35/vendor/github.com/openshift/library-go/pkg/operator/status/condition.go#L68-L71